test/backend/datastores.athena.spec.js

```diff
@@ -0,0 +1,217 @@
+import {assert} from 'chai';
+
+import {connect} from '../../backend/persistent/datastores/athena';
+
+/* eslint-disable no-invalid-this */
+describe('Athena Unit Tests:', function () {
+    it('Invalid connection strings because incorrect credentials', function(done) {
+        const conn = {
+            region: 'us-east-1',
+            accessKey: 'XXXXXXXX',
+            secretKey: 'XXXXXAAAA',
+            database: 'PLOT.LY-TEST',
+            sqlStatement: 'SELECT * FROM TEST_TABLE LIMIT 100',
+            s3Outputlocation: 's3://aws-athena-query-results-11111111-us-east-1/',
+            queryTimeout: 10000
+        };
+        connect(conn).then(connection => {
+           done(`Should NOT have obtained a valid connection ${connection}`);
+        }).catch(err => {
+            assert.isDefined(err, 'Connection failed due to incorrect connection string');
```

Comment:
the assert msg should describe why the test failed (in this because err isn't defined).

we could make the test tighter, e.g.:
```js
        assert.equal(err.name, '...', 'Unexpected error type');
        assert.equal(err.message, '...', 'Unexpected error message');
```